composer.json

```diff
@@ -57,13 +57,13 @@
     "php": "^7.1",
     "clearfw/clearfw": "~1.2.0",
     "easyrdf/easyrdf": "^1.1",
-    "doctrine/dbal": "2.10.1",
+    "doctrine/dbal": "^2.10.1",
     "doctrine/annotations": "~1.6.0",
     "laminas/laminas-servicemanager": "~2.5.0",
     "league/flysystem": "~1.0",
     "league/flysystem-memory": "~1.0",
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-    "oat-sa/lib-generis-search": "2.1.2",
+    "oat-sa/lib-generis-search": "^2.1.2",
```

Comment:
if i remember correctly this versioning means >= 2.1.2 && <2.1.3, is it initial idea for of this change? is this restriction make sense?

Reply:
I'm pretty sure it means `>=2.1.2 <3.0.0-0`.

Reply:
https://jubianchi.github.io/semver-check/#/^2.1.2/2.9.9